docs/integrations/valtio.mdx

```diff
@@ -70,3 +70,51 @@ atomWithProxy(proxyObject, { sync: true })
 ### Examples
 
 <CodeSandbox id="ew98ll" />
+
+## mutableAtom
+
+`mutableAtom` wraps a value in a self-aware Valtio proxy. You can make changes to it in the same way you would to a normal js-object.
+
+Count value is stored under the `value` property.
+
+```jsx
+const countProxyAtom = mutableAtom(0)
+
+function IncrementButton() {
+  const countProxy = useAtomValue(countProxyAtom)
+  return <button onClick={() => countProxy.value++}>+</button>
+}
+```
+
+### Parameters
+
+```js
+mutableAtom(value, options?)
+```
+
+**value** (required): the value to proxy.
+
+**options.proxyFn** (optional): allows customization with `proxyFn` for custom proxy functions. Can be `proxy` (default) or a custom function.
+
+### Examples
+
+<CodeSandbox id="f84sk5" />
```

Comment:
What is needed to get this to work?

Full URL: https://codesandbox.io/s/morning-darkness-f84sk5

Reply:
What do you mean?

It seems working: https://jotai-git-fork-dmaskasky-docs-valtio-mutableatom-pmndrs.vercel.app/docs/integrations/valtio

Reply:
Oh nice.